src/main/scala/com.socrata.tileserver/util/TileEncoder.scala

```diff
@@ -45,12 +50,26 @@ case class TileEncoder(features: Set[TileEncoder.Feature]) {
   /** Create a vector tile as a base64 encoded protocol-buffer. */
   lazy val base64: String = Base64.encodeBase64String(bytes)
 
-  /** Create a Seq of Well Known Binary geometries. */
-  lazy val wkbs: Map[String, Seq[Array[Byte]]] = {
+  /** Create a Seq of Well Known Binary geometries and attributes. */
+  lazy val wkbsAndAttributes: Map[String, Seq[Map[String, String]]] = {
     val grouped = features.toSeq.groupBy { case (geom, _) => layerName(geom) }
 
     grouped.map { case (layer, features) =>
-      layer -> features.map { case (geom, _) => writer.write(geom) }
+      layer -> features.map {
+        case (geom: Geometry, attributes) => Map(
+          "wkbs" -> Base64.encodeBase64String(writer.write(geom)),
```

Comment:
I'm actually not sure if this even needs to be Base64 encoded anymore, I think this might be a from when we used JSON instead of MsgPack.